bitcoind-rpc/src/main/scala/org/bitcoins/rpc/jsonmodels/RpcPsbtResult.scala

```diff
@@ -69,3 +70,25 @@ final case class WalletCreateFundedPsbtResult(
     fee: Bitcoins,
     changepos: Int
 ) extends RpcPsbtResult
+
+
+final case class AnalyzePsbtResult(
+                                  inputs: Vector[AnalyzePsbtInput],
+                                  estimated_vsize: Option[Double],
+                                  estimated_feerate: Option[Double],
+                                  fee: Option[Bitcoins],
+                                  next: String
```

Comment:
I don't really understand what this next field is for?

Reply:
Is there a fixed list of people in "next"? Do we know what the possible values are?

Reply:
See that's what I am unsure about. Here is the text provided when you run `help` in `cli` for `analyzepsbt` 
```
 "next" : "role"             (string, optional) Role of the next person that this input needs to go to
```

Reply:
So the PSBT "life-cycle" consists of different people with different roles. See this doc for more: https://github.com/bitcoin/bitcoin/blob/master/doc/psbt.md#overall-workflow

Here's an example of what this looks like: 

```shell
❯ bitcoin-cli analyzepsbt   cHNidP8BAHUCAAAAASaBcTce3/KF6Tet7qSze3gADAVmy7OtZGQXE8pCFxv2AAAAAAD+////AtPf9QUAAAAAGXapFNDFmQPFusKGh2DpD9UhpGZap2UgiKwA4fUFAAAAABepFDVF5uM7gyxHBQ8k0+65PJwDlIvHh7MuEwAAAQD9pQEBAAAAAAECiaPHHqtNIOA3G7ukzGmPopXJRjr6Ljl/hTPMti+VZ+UBAAAAFxYAFL4Y0VKpsBIDna89p95PUzSe7LmF/////4b4qkOnHf8USIk6UwpyN+9rRgi7st0tAXHmOuxqSJC0AQAAABcWABT+Pp7xp0XpdNkCxDVZQ6vLNL1TU/////8CAMLrCwAAAAAZdqkUhc/xCX/Z4Ai7NK9wnGIZeziXikiIrHL++E4sAAAAF6kUM5cluiHv1irHU6m80GfWx6ajnQWHAkcwRAIgJxK+IuAnDzlPVoMR3HyppolwuAJf3TskAinwf4pfOiQCIAGLONfc0xTnNMkna9b7QPZzMlvEuqFEyADS8vAtsnZcASED0uFWdJQbrUqZY3LLh+GFbTZSYG2YVi/jnF6efkE/IQUCSDBFAiEA0SuFLYXc2WHS9fSrZgZU327tzHlMDDPOXMMJ/7X85Y0CIGczio4OFyXBl/saiK9Z9R5E5CVbIBZ8hoQDHAXR8lkqASECI7cr7vCWXRC+B3jv7NYfysb3mk6haTkzgHNEZPhPKrMAAAAAAAAA
{
  "inputs": [
    {
      "has_utxo": true,
      "is_final": false,
      "missing": {
        "pubkeys": [
          "85cff1097fd9e008bb34af709c62197b38978a48"
        ]
      },
      "next": "updater"
    }
  ],
  "fee": 0.00000301,
  "next": "updater"
}

```

So I have a hunch that the different roles can be `creator`, `updater`, `signer`, `finalizer, `extractor` and `combiner`